tests/models/test_truncated_bptt.py

```diff
@@ -0,0 +1,97 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
Since you created this file, can you also move the test `test_tbptt_log` in `tests/trainer/logging_/test_train_loop_logging.py`?